src/Config.php

```diff
@@ -4,22 +4,22 @@
 
 use FINDOLOGIC\Api\Exceptions\ConfigException;
 use FINDOLOGIC\Api\Validators\ConfigValidator;
-use FINDOLOGIC\GuzzleHttp\Client;
+use GuzzleHttp\Client;
 
 class Config
 {
-    const
+    public const
```

Comment:
This changes regarding public constants were added during the version bump of the linting library